eva-accession-release/src/main/java/uk/ac/ebi/eva/accession/release/io/MergedVariantMongoReader.java

```diff
@@ -135,11 +138,21 @@ private String getInactiveField(String field) {
 
                 addToVariants(variants, contig, submittedVariantStart, rs, reference, alternate, sourceEntry);
             }
+
+            if (submittedVariantOperation.getString(EVENT_TYPE_FIELD).equals(EventType.UPDATED.toString())
+                    && submittedVariantOperation.getString(REASON).startsWith(DECLUSTER_OPERATION_REASON_FIRST_WORD)) {
+                hasSubmittedVariantsDeclustered = true;
+            }
         }
 
-        if (variants.isEmpty()) {
-            throw new IllegalStateException("There was a merge operation for rs" + rs + " but there is no update " +
-                                                    "operation for the SS IDs.");
+        if (!hasSubmittedVariantsDeclustered && variants.isEmpty()) {
+            throw new IllegalStateException ("There was a merge operation for rs" + rs + " but there is no update " +
```

Comment:
Suggested simplification of this message:
> "Found merge operation for rs" + rs + " but no SS IDs updates (merge/update) in the collection containing operations. Since every RS ID in dbSNP is associated with at least one SS ID, the latter must be updated when the former are merged, unless they have been previously declustered (RS ID = null)."